comprehensive/lms/templates/user_dropdown.html

```diff
@@ -20,9 +20,7 @@
             <span class="sr-only">${_("Dashboard for:")}</span>
             <%
             username = self.real_user.username
-            profile_image_url = get_profile_image_urls_for_user(self.real_user)['small']
             %>
-            <!--<img class="menu-image" src="${profile_image_url}" alt="">-->
```

Comment:
nit: you can remove the import of get_profile_image_urls_for_user above, but this is optional